<!DOCTYPE html>
<html lang="en"> 
<head
	<meta name="google-site-verification" content="ZHoTRPqjl4B81gIH3Z6oowjB1XPgrPgeAaUpmXUJAFI" />
	<title>BAYSM 2025 - The Bayesian Young Statisticians Meeting</title>
	
	<!-- Meta -->
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="description" content="The Bayesian Young Statisticians Meeting">
	<meta name="author" content="template made by: Xiaoying Riley at 3rd Wave Media">    
	<link rel="shortcut icon" href="bayesm.ico"> 
	
	<!-- Google Font -->
	<link href="https://fonts.googleapis.com/css?family=Montserrat:600,700,800|Roboto:300,400,700&display=swap" rel="stylesheet">
	
	<!-- FontAwesome JS-->
	<script defer src="assets/fontawesome/js/all.min.js"></script>

	<!-- Theme CSS -->  
	<link id="theme-style" rel="stylesheet" href="assets/css/theme.css">


</head> 

<body>    
	<header id="header" class="header fixed-top">	    
		<div class="branding">
			<div class="container-fluid">
				<nav class="main-nav navbar navbar-expand-lg">   
					
					<div class="navbar-btn order-lg-2"><a class="btn btn-primary btn-lg" href="#tickets-section" target="_blank">Register</a></div>
					
					<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navigation" aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
						<span class="navbar-toggler-icon"></span>      
					</button>
					
					<div id="navigation" class="navbar-collapse collapse  justify-content-lg-end me-lg-3">
						<ul class="nav navbar-nav">
							
							<li class="nav-item"><a class="nav-link scrollto" href="#about-section">About</a></li>                                              
							<li class="nav-item"><a class="nav-link scrollto" href="#speakers-section">Keynote Speakers</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#BRspeakers-section">BR Award Speakers</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#schedule-section">Schedule</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#abstract-section">Abstract Submission</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#tickets-section">Registration</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#venue-section">Committees</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#contacts-section">Contacts</a></li>
							<li class="nav-item"><a class="nav-link scrollto" href="#sponsors-section">Sponsors</a></li>
						</ul><!--//nav-->
					</div><!--//navabr-collapse-->

				</nav><!--//main-nav-->
				
			</div><!--//container-->
		</div><!--//branding-->
	</header><!--//header-->
	
	<div id="hero-block" class="hero-block">
		<div id="hero-carousel" class="hero-carousel carousel slide carousel-fade" data-ride="carousel">
			<div class="carousel-inner">
				<div class="carousel-item-1 carousel-item active">
				</div>
				<div class="carousel-item-2 carousel-item">
				</div>
				<div class="carousel-item-3 carousel-item">
				</div>
			</div>
		</div>
		<div class="hero-block-mask"></div>
		<div class="container">
			<div class="hero-text-block">
				<h1 class="hero-heading mb-2">BAYSM 2025</h1>
				<div class="hero-meta mb-3"><i class="far fa-calendar-alt me-2"></i>Dates: 7-11 April 2025 <i class="fas fa-map-marker-alt mx-2"></i>Online Event</div>
				<div class="hero-intro mb-4">The Bayesian Young Statisticians Meeting</div>
				<div class="hero-cta"><a class="btn btn-primary btn-lg" href="#tickets-section" target="_blank">Register</a>
				<a href="https://forms.gle/85XpcHJBkPXYeNsB8" class="btn btn-secondary btn-lg d-block d-md-inline-block" target="_blank">Submit Abstract</a></div>
				
			</div><!--//hero-text-block-->
		</div><!--//container-->

	</div><!--//hero-block-->
	
	<div class="stats-block theme-bg-primary text-white py-4 text-center">
		<div class="container">
			<div class="row">
				<div class="col-6 col-md-3">
					<div class="item">
						<div class="number">5</div>
						<div class="unit">Days</div>
					</div><!--//item-->
				</div><!--//col-->
				<div class="col-6 col-md-3">
					<div class="item">
						<div class="number">5</div>
						<div class="unit">Keynote Speakers</div>
					</div><!--//item-->
				</div><!--//col-->
				<div class="col-6 col-md-3">
					<div class="item">
						<div class="number">6</div>
						<div class="unit">Talks of Blackwell-Rosenbluth Award Winners</div>
					</div><!--//item-->
				</div><!--//col-->
				<div class="col-6 col-md-3">
					<div class="item">
						<div class="number">TBA</div>
						<div class="unit">Talks</div>
					</div><!--//item-->
				</div><!--//col-->
			</div>
		</div><!--//container-->
	</div><!--//stats-block-->

	<section id="about-section" class="about-section section theme-bg-light">
		<div class="container">
			<h3 class="section-heading text-center mb-3">About BAYSM</h3>
			<div class="section-intro single-col-max mx-auto mb-4"> 
			<p align="justify">
			BAYSM 2025 is the 9th edition of the Bayesian Young Statisticians Meeting: the official conference of <a href="https://j-isba.github.io/"><b>j-ISBA</b></a>, the junior section of the International Society for Bayesian Analysis (<a href="https://bayesian.org"><b>ISBA</b></a>). <br>

			BAYSM is held in person in even-numbered years, typically in conjunction with the ISBA World Meeting, and online in odd-numbered years. <b>BAYSM 2025 will be a fully online conference</b>.<br>
				
			BAYSM is a platform for students and early career researchers to interconnect with the Bayesian community. 
			It is an opportunity to get in touch with established senior researchers and start building a knowledge network. <br>
			
			The platform encourages discussion and promotes research in a wide spectrum of fields where Bayesian methods are employed.<br>
			</p>
			<p>
			All attendees are expected to adhere to the <a href="https://bayesian.org/governance/code-of-conduct/"><b>ISBA code of conduct</b></a> 
			and related policies provided by <a href="https://bayesian.org/membership/safe-isba/"><b>safeISBA</b></a>.
			</p>
			<p>
			A recollection of all previous BAYSM editions can be found <a href="https://baysmjisba.github.io/"><b>here</b></a>.
			</p>
				
			<div class="event-countdown text-center mb-3">		   
				<h4 class="countdown-intro mb-2 text-center mb-3">Event Starts In:</h4>
				<div id="countdown-box" class="countdown-box"></div>
			</div><!--//event-countdown-->
	</section><!--//about-section-->
	
	<section id="speakers-section" class="speakers-section section">
		<div class="container">
			<h3 class="section-heading text-center mb-3">Keynote Speakers</h3>
			<div class="section-intro text-center single-col-max mx-auto mb-5"></div>
			<div class="row">
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-1" data-bs-toggle="modal" data-bs-target="#modal-speaker-1"><img src="assets/images/speakers/speaker-1.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Samuel Livingstone</h5>
							<div class="card-text mb-3">
								<div class="meta">University College London (UCL)</div>
								<div class="meta"><a href="https://samueljlivingstone.wixsite.com/webpage"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Robust sampling from irregular posterior distributions. Some thoughts from a decade of trying.</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-1" data-bs-toggle="modal" data-bs-target="#modal-speaker-1">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
					
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-2" data-bs-toggle="modal" data-bs-target="#modal-speaker-2"><img src="assets/images/speakers/speaker-2.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Julia Palacios</h5>
							<div class="card-text mb-3">
								<div class="meta">Stanford University</div>
								<div class="meta"><a href="https://juliapalacios.github.io/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Novel coalescent point process approaches for Bayesian inference of disease and molecular dynamics. </b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-2" data-bs-toggle="modal" data-bs-target="#modal-speaker-2">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
					
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-3" data-bs-toggle="modal" data-bs-target="#modal-speaker-3"><img src="assets/images/speakers/speaker-3.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Fernando Quintana</h5>
							<div class="card-text mb-3">
								<div class="meta">Pontificia Universidad Católica de Chile</div>
								<div class="meta"><a href="https://www.mat.uc.cl/~quintana/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: On a Class of Repulsive Mixture Models: Some results and applications</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-3" data-bs-toggle="modal" data-bs-target="#modal-speaker-3">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-6 col-lg-2 mb-4">
				</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-4" data-bs-toggle="modal" data-bs-target="#modal-speaker-4"><img src="assets/images/speakers/speaker-4.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Surya Tokdar</h5>
							<div class="card-text mb-3">
								<div class="meta">Duke University</div>
								<div class="meta"><a href="http://www2.stat.duke.edu/~st118/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Not Your Average Statistical Analysis</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-4" data-bs-toggle="modal" data-bs-target="#modal-speaker-4">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-5" data-bs-toggle="modal" data-bs-target="#modal-speaker-5"><img src="assets/images/speakers/speaker-5.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Stéphanie van der Pas</h5>
							<div class="card-text mb-3">
								<div class="meta">Vrije Universiteit Amsterdam</div>
								<div class="meta"><a href="https://www.stephanievanderpas.nl/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Causal conclusions from a cutoff - Bayesian regression discontinuity designs </b>
							</div><!--//card-text-->
							<a href="#modal-speaker-5" data-bs-toggle="modal" data-bs-target="#modal-speaker-5">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
			</div><!--//row-->
		</div><!--//container-->
		
	</section><!--//speakers-section-->

	<section id="BRspeakers-section" class="speakers-section section theme-bg-light">
		<div class="container">
			<h3 class="section-heading text-center mb-3">Blackwell-Rosenbluth Award Talks</h3>
			<div class="section-intro text-center single-col-max mx-auto mb-5"></div>
			<div class="row">
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-BR1" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR1"><img src="assets/images/fan.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Fan Bu</h5>
							<div class="card-text mb-3">
								<div class="meta">University of Michigan</div>
								<div class="meta"><a href="https://fanbu1995.github.io/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Data Augmentation Approaches for Inference of Partially Observed Epidemics with Individual Heterogeneity</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-1" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR1">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
					
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-BR2" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR2"><img src="assets/images/raaz.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Raaz Dwivedi</h5>
							<div class="card-text mb-3">
								<div class="meta">Cornell University</div>
								<div class="meta"><a href="https://raazdwivedi.github.io/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Advances in Distribution Compression and Its Consequences</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-BR2" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR2">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
					
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-BR3" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR3"><img src="assets/images/qian.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Qian Qin</h5>
							<div class="card-text mb-3">
								<div class="meta">University of Minnesota</div>
								<div class="meta"><a href="https://qqin.umn.edu/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Spectral gap bounds for reversible hybrid Gibbs chains</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-BR3" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR3">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-BR4" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR4"><img src="assets/images/giovanni.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Giovanni Rebaudo</h5>
							<div class="card-text mb-3">
								<div class="meta">University of Turin</div>
								<div class="meta"><a href="https://giovannirebaudo.github.io/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Graph-Aligned Random Partition Model (GARP)</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-BR4" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR4">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-BR5" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR5"><img src="assets/images/tommaso.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Tommaso Rigon</h5>
							<div class="card-text mb-3">
								<div class="meta">University of Milano-Bicocca</div>
								<div class="meta"><a href="https://tommasorigon.github.io/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Bayesian analysis of product feature allocation models</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-BR5" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR5">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					<div class="card rounded-0">
						<a href="#modal-speaker-BR6" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR6"><img src="assets/images/francesca.jpg" class="card-img-top rounded-0" alt=""></a>
						<div class="card-body">
							<h5 class="card-title mb-2">Francesca Romana Crucinio</h5>
							<div class="card-text mb-3">
								<div class="meta">King’s College London (at the time of the award), University of Turin (current affiliation)</div>
								<div class="meta"><a href="https://francescacrucinio.github.io/"><b>Personal website</b></a></div>
								<div class="meta"><b>Title: Towards a turnkey approach to unbiased Monte Carlo estimation of smooth functions of expectations (with applications to exponential random graphs)</b></div>
							</div><!--//card-text-->
							<a href="#modal-speaker-BR6" data-bs-toggle="modal" data-bs-target="#modal-speaker-BR6">Abstract &rarr;</a>
						</div><!--//card-->
					</div><!--//card-->
				</div><!--//col-->
			</div><!--//row-->
		</div><!--//container-->
		
	
	<section id="schedule-section" class="schedule-section section">
		<div class="container">
			<h3 class="section-heading text-center mb-5">Schedule in UTC+0</h3>

			<!-- Nav tabs -->
			<ul class="schedule-nav nav nav-pills nav-fill" id="myTab" role="tablist">
				<li class="nav-item me-2">
					<a class="nav-link active" id="tab-1" data-bs-toggle="tab" href="#tab-1-content" role="tab" aria-controls="tab-1-content" aria-selected="true">
						<span class="heading">Day 1</span>
						<span class="meta">(TBA)</span>
					</a>
				</li>
				<li class="nav-item me-2">
					<a class="nav-link" id="tab-2" data-bs-toggle="tab" href="#tab-2-content" role="tab" aria-controls="tab-2-content" aria-selected="false">
						<span class="heading">Day 2</span>
						<span class="meta">(TBA)</span>
					</a>
				</li>
				<li class="nav-item">
					<a class="nav-link" id="tab-3" data-bs-toggle="tab" href="#tab-3-content" role="tab" aria-controls="tab-3-content" aria-selected="false">
						<span class="heading">Day 3</span>
						<span class="meta">(TBA)</span>
					</a>
				</li>
				<li class="nav-item">
					<a class="nav-link" id="tab-4" data-bs-toggle="tab" href="#tab-4-content" role="tab" aria-controls="tab-4-content" aria-selected="false">
						<span class="heading">Day 4</span>
						<span class="meta">(TBA)</span>
					</a>
				</li>
				<li class="nav-item">
					<a class="nav-link" id="tab-5" data-bs-toggle="tab" href="#tab-5-content" role="tab" aria-controls="tab-5-content" aria-selected="false">
						<span class="heading">Day 5</span>
						<span class="meta">(TBA)</span>
					</a>
				</li>
			</ul>
			
			<!-- Tab panes -->
			<div class="schedule-tab-content tab-content">
				<div class="tab-pane no-timeline" id="tab-1-content" role="tabpanel" aria-labelledby="tab-1">
					<h4 class="text-center py-5 text-muted">Day 1 Agenda Coming Soon...</h4>
				</div><!--//tab-2-content-->
				<div class="tab-pane no-timeline" id="tab-2-content" role="tabpanel" aria-labelledby="tab-2">
					<h4 class="text-center py-5 text-muted">Day 2 Agenda Coming Soon...</h4>
				</div><!--//tab-2-content-->
				<div class="tab-pane no-timeline" id="tab-3-content" role="tabpanel" aria-labelledby="tab-3">
					<h4 class="text-center py-5 text-muted">Day 3 Agenda Coming Soon...</h4>
				</div><!--//tab-3-content-->
				<div class="tab-pane no-timeline" id="tab-4-content" role="tabpanel" aria-labelledby="tab-4">
					<h4 class="text-center py-5 text-muted">Day 4 Agenda Coming Soon...</h4>
				</div><!--//tab-4-content-->
				<div class="tab-pane no-timeline" id="tab-5-content" role="tabpanel" aria-labelledby="tab-5">
					<h4 class="text-center py-5 text-muted">Day 5 Agenda Coming Soon...</h4>
				</div><!--//tab-5-content-->
			</div><!--//schedule-tab-content-->
			<div class="schedule-cta text-center mx-auto"><a href="https://baysm2025.github.io/" class="btn btn-primary btn-lg me-md-2 d-block d-md-inline-block mb-3 mb-md-0" target="_blank">Download the schedule in your time zone</a></div>
		</div><!--//container-->
	</section><!--//schedule-section-->

	<section id="abstract-section" class="abstract-section section theme-bg-light">
		<div class="container">
			<h3 class="section-heading text-center mb-3">Abstract submission</h3>
			<div class="section-intro single-col-max mx-auto mb-4"> 
				We invite contributions on various topics related to Bayesian theory, methods, and applications. Please submit an extended abstract of 700 to 1,000 words in PDF format. 
				<br>
				Please adhere to the specified length to ensure full consideration of your submission.
				<br>
				<b>Abstract submission is closed (deadline: January 15, 2025).</b>
				<br>
				Notification of acceptance by March 3, 2025.
				<br>
			</div>
		</div>
	</section><!--//about-section-->
				
	<section id="tickets-section" class="tickets-section section theme-bg-light">
		<div class="container">
			<h3 class="section-heading text-center mb-3">Registration</h3>
			<div class="section-intro single-col-max mx-auto mb-4"> 
			<p align="justify">
				<b>Registration Deadline: 31 March 2025</b> <br>
				<b>Instructions:</b> To register, click the "Register" button below and complete the form. <br>
				If you wish to apply the j-ISBA member discount, you must first visit <a href="https://bayesian.org">bayesian.org </a> and log in 
				(the login button is in the top right corner of the homepage). After logging in, return to the BAYSM2025 website and click the "Register" button below.<br>
				Please ensure that the "Primary Email Address" field in the registration form is correct, as this is where you will receive both the payment receipt and 
				the access link for the full conference. Access links are personal and cannot be shared with others.<br>
				If you encounter any issues, please do not hesitate to contact the conference organizing committee at <a href="mailto:baysm2025@gmail.com"><i>baysm2025@gmail.com </i></a>.
			</p>
			</div>
			<div class="row pricing mb-5">
				<div class="col-12 col-md-4 p-2 p-lg-4">
					<div class="card rounded-0 border-0">
						<div class="card-body p-0">
							<div class="heading  text-center p-3">
								<h4 class="text-white mb-0">j-ISBA members</h4>
							</div>
							<div class="info p-3">
								<div class="price-figure text-center mb-3"><span class="currency">$</span><span class="number">5 USD</span></div>
								<div class="desc px-3">
									<b>How to become a j-ISBA member</b> <br>
									To become a j-ISBA member, you must first be an ISBA member. <br>
									If you are already an ISBA member but not yet a j-ISBA member, you can enroll in j-ISBA <a href="https://bayesian.org/civicrm/contribute/transact/?reset=1&id=36"><i>here</i></a>. <br>
									If you are not an ISBA member, you can enroll in both ISBA and j-ISBA <a href="https://bayesian.org/membership/joinrenew/"><i>here</i></a>. <br>
									Please note that early career researchers who have graduated from a degree program in the last 5 years can enjoy the reduced ISBA fee.
								</div>
							</div><!--//info-->
						</div><!--//card-body-->
						<div class="card-footer text-center">
							<a class="btn btn-primary btn-lg" href="https://bayesian.org/civicrm/event/register/?reset=1&id=68" target="_blank">Register</a>
						</div>
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-12 col-md-4 p-2 p-lg-4">
					<div class="card card-special rounded-0 border-0">
						<div class="card-body p-0">
							<div class="heading  text-center p-3">
								<h4 class="text-white mb-0">Regular</h4>
							</div>
							<div class="info p-3">
								<div class="price-figure text-center"><span class="currency">$</span><span class="number">50 USD</span></div>
								
							</div><!--//info-->
						</div><!--//card-body-->
						<div class="card-footer text-center">
							<a class="btn btn-primary btn-lg" href="https://bayesian.org/civicrm/event/register/?reset=1&id=68" target="_blank">Register</a>
						</div>
					</div><!--//card-->
				</div><!--//col-->
				<div class="col-12 col-md-4 p-2 p-lg-4">
					<div class="card rounded-0 border-0">
						<div class="card-body p-0">
							<div class="heading  text-center p-3">
								<h4 class="text-white mb-0"> Fee exempted</h4>
							</div>
							<div class="info p-3">
								<div class="price-figure text-center mb-3"><span class="currency">$</span><span class="number">0 USD</span></div>
								<div class="desc px-3">
									People who reside and work in countries with a per capita GNI not exceeding $6,000 per person are exempt from the fee.<br>
									Registration is mandatory for everyone and required to receive the link to attend the conference.					    
								</div>
							</div><!--//info-->
						</div><!--//card-body-->
						<div class="card-footer text-center">
							<a class="btn btn-primary btn-lg" href="https://bayesian.org/civicrm/event/register/?reset=1&id=68" target="_blank">Register</a>
						</div>
					</div><!--//card-->
				</div><!--//col-->
			</div><!--//pricing-->
		</div><!--//container-->
	</section><!--//tickets-section-->
	
	<section id="contacts-section" class="contacts-section section theme-bg-light">
		<div class="container">
			<h3 class="section-heading text-center mb-3">Contacts</h3>
			<div class="section-intro single-col-max mx-auto mb-4"> 
				For any questions, please write to <a href="mailto:baysm2025@gmail.com"><i>baysm2025@gmail.com </i></a></li>. 
				<br>
				<br>
				j-ISBA website: <a href="https://j-isba.github.io">https://j-isba.github.io</a><br>
			<br>
j-ISBA twitter: <a href="https://twitter.com/j_ISBA">https://twitter.com/j_ISBA</a><br>
			<br>
			</div>
		</div>
	</section><!--//about-section-->
				
	<section id="venue-section" class="venue-section section  theme-bg-primary text-white">
		<div class="container">
			<h3 class="section-heading text-center mb-5 text-white">Committees</h3>
			<div class="row gx-5 py-lg-5">
				<div class="col-6 col-lg-4 mb-4">
					<div class="desc">
						<h4 class="text-white mb-3">Scientific committee </h4>
						<p>	<b>Filippo Ascolani</b>, Duke Univeristy <br>
							<b>Elizabeth Bersson</b>, Massachusetts Institute of Technology <br>
							<b>Ayush Bharti</b>, Aalto University <br>
							<b>Jordan Bryan</b>, University of North Carolina <br>
							<b>Noirrit Kiran Chandra</b>, The University of Texas at Dallas <br>
							<b>Francesca Romana Crucinio</b>, University of Turin <br>
							<b>Yunshan Duan</b>, Univesity of Texas at Austin <br>
							<b>Beatrice Franzolini</b>, Bocconi University <br>
							<b>Matteo Giordano</b>, University of Turin <br>
							<b>Jack Jewson</b>, Monash University <br>
							<b>Kyeongwon Lee</b>, University of Maryland <br>
							<b>Ilsang Ohn</b>, Inha University <br>
							<b>Sally Paganin</b>, The Ohio State University <br>
							<b>Francesca Panero</b>, Sapienza University <br>
							<b>Paul Rosa</b>, University of Oxford <br>
							<b>Angélica M. Rosario Santos</b>, University of Puerto Rico <br>
							<b>Michael Minyi Zhang</b>, University of Hong Kong <br>
							<b>Alessandro Zito</b>, Harvard University<br>
						</p>
						</div><!--//desc-->
				</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					</div><!--//col-->
				<div class="col-6 col-lg-4 mb-4">
					<div class="desc">
						<h4 class="text-white mb-4 mt-3 mt-lg-5">Organizing committee</h4>
						<p>	<b>Filippo Ascolani</b>, Duke Univeristy <br>
							<b>Jordan Bryan</b>, University of North Carolina <br> 
							<b>Beatrice Franzolini</b>, Bocconi University <br>
							<b>Matteo Giordano</b>, University of Turin <br>
							<b>Beniamino Hadj-Amar</b>, Rice University <br>
							<b>Francesca Panero</b>, Sapienza University <br>
							<b>Alessandro Zito</b>, Harvard University<br>
						</p>
					</div><!--//desc-->
				</div><!--//col-->
			</div><!--//row-->
		</div><!--//container-->
	</section><!--//venue-section-->

		<section id="sponsors-section" class="sponsors-section section">
		<div class="container">
			<h3 class="section-heading text-center mb-3">Sponsors</h3>
			<div class="section-intro text-center single-col-max mx-auto mb-5">Want to become a sponsor? Get in touch with j-ISBA at <a href="mailto:baysm2025@gmail.com"><i>jisba.section@gmail.com</i></a></li>.</div>
			<div class="row logos justify-content-center">
				<div class="logo-item col-6 col-md-4 col-lg-2"><img src="assets/images/logo-1.jpg" alt=""></div>
				<div class="logo-item col-6 col-md-4 col-lg-2"><img src="assets/images/logo-2.png" alt=""></div>
			</div><!--//row-->
		</div><!--//container-->
	</section><!--//sponsors-section-->
				
	<footer class="footer py-5 theme-bg-primary">
		<div class="container text-center">
			
			<ul class="social-list list-inline mb-4"> 
				<li class="list-inline-item me-3"><a href="mailto:baysm2025@gmail.com"><i class="fas fa-envelope"></i></a></li>
				<li class="list-inline-item me-3"><a href="https://x.com/j_isba?lang=en"><i class="fab fa-twitter fa-fw"></i></a></li>
			</ul><!--//social-list-->
			
			
			<ul class="footer-links list-inline mx-auto mb-4"> 
				<li class="list-inline-item"><a href="https://bayesian.org/governance/code-of-conduct/">Code of Conduct</a></li>
			</ul><!--//footer-link-->
			
			 <!--/* This template is free as long as you keep the footer attribution link. If you'd like to use the template without the attribution link, you can buy the commercial license via our website: themes.3rdwavemedia.com Thank you for your support. :) */-->
			<small class="copyright">Designed with <i class="fas fa-heart" style="color: #EC645E;"></i> by <a href="http://themes.3rdwavemedia.com" target="_blank">Xiaoying Riley</a> for developers</small>
			
		</div><!--//container-->	    
	</footer>

	<!-- Modal Speaker -->
	<div class="modal modal-speaker modal-speaker-1" id="modal-speaker-1" tabindex="-1" role="dialog" aria-labelledby="speaker-1-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-1-ModalLabel" class="modal-title sr-only">Samuel Livingstone</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/speakers/speaker-1.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Samuel Livingstone</h2>
								<div class="meta">Associate Professor</div>
								<div class="meta mb-2">University College London (UCL)</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Robust sampling from irregular posterior distributions. Some thoughts from a decade of trying. </p>
						<p class="mb-0">Abstract: Often Bayesian modeling strategies can result in posterior distributions with irregular features, such as severe anisotropies, very light tails or a high level of skew.  In these scenarios state-of-the-art gradient-based sampling algorithms based on Langevin and Hamiltonian dynamics can become unstable, but there are some simple adjustments either to the process or the numerical scheme used to simulate it that can stabilise behaviour in a quantifiable way.  I will discuss several strategies that can be employed to deal with these kinds of problems, based on my own work on robust sampling over the last decade.  Examples include changing the kinetic energy in Hamiltonian Monte Carlo, robust numerical integrators for Langevin dynamics using skew-symmetric increments, and strategies for pre-conditioning algorithms to speed up mixing.  Depending on time constraints I will aim to discuss the key ideas behind each of these at an accessible level.  This is based on joint work with many fantastic students and collaborators, too many to name here, but I will be sure to mention them all in the talk.</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker -->
	<div class="modal modal-speaker modal-speaker-2" id="modal-speaker-2" tabindex="-1" role="dialog" aria-labelledby="speaker-2-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-2-ModalLabel" class="modal-title sr-only">Julia Palacios</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/speakers/speaker-2.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Julia Palacios</h2>
								<div class="meta">Associate Professor</div>
								<div class="meta mb-2">Stanford University</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Novel coalescent point process approaches for Bayesian inference of disease and molecular dynamics. </p>
						<p class="mb-0">Abstract: Coalescent point processes are used to infer population dynamics at different time scales from molecular sequence data. These methods have been applied to study cell population dynamics during developmental processes, to infer disease transmission dynamics of rapidly evolving pathogens, and to infer ancestral human population size trajectories. In this talk I will describe two challenging situations for coalescent inference of disease dynamics and cell dynamics. In the former, we derive a coalescent model for exposed-infected (E-I) population dynamics that allows us to gain parameter interpretability by directly estimating the number of infected individuals over time and the effective reproduction number from a sample genealogy. In the latter, we use a bounded coalescent point process conditioning on all events happening within a certain time interval. We develop an exact inferential framework for this problem that does not rely on large data augmentation or approximations of the likelihood function.</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker -->
	<div class="modal modal-speaker modal-speaker-3" id="modal-speaker-3" tabindex="-1" role="dialog" aria-labelledby="speaker-3-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-3-ModalLabel" class="modal-title sr-only">Fernando Quintana</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/speakers/speaker-3.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Fernando Quintana</h2>
								<div class="meta">Full Professor</div>
								<div class="meta mb-2">Pontificia Universidad Católica de Chile</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: On a Class of Repulsive Mixture Models: Some results and applications </p>
						<p class="mb-0">Abstract: Many random partition models, including those induced from mixtures of several popular discrete random probability measures, use iid-generated parameters to define clusters. It is common in this context to find overlapping clusters or even some that are too small. To alleviate these problems, some repulsive mixture models have been proposed. This talk reviews some of these methods and focuses on one specific such construction, discusses some of its properties and presents some applications to density estimation and clustering.</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

				<!-- Modal Speaker -->
	<div class="modal modal-speaker modal-speaker-4" id="modal-speaker-4" tabindex="-1" role="dialog" aria-labelledby="speaker-4-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-4-ModalLabel" class="modal-title sr-only">Surya Tokdar</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/speakers/speaker-4.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Surya Tokdar</h2>
								<div class="meta">Full Professor</div>
								<div class="meta mb-2">Duke University</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Not Your Average Statistical Analysis </p>
						<p class="mb-0">Abstract: Statistical analyses of heavy tailed data bring in a unique set of questions. 
							Often the scientific focus shifts to the tails of the distribution, e.g., to forecasting the 100-year daily precipitation or to identifying predictors which influence extreme low birthweight. 
							Parametric models – with their fit largely dictated by the central bulk of the data – may not do justice to capturing tail structures. 
							At the same time, purely nonparametric approaches may prove futile in effectively smoothing information from sparse observations across an elongated tail. 
							Toward more effective statistical analyses of heavy tailed data, I will introduce a class of semiparametric Bayesian methods for density estimation and quantile regression. 
							With a carefully chosen nonparametric prior distribution, the density estimation method will be shown to simultaneously guarantee accurate estimation of the density function and its tail index, both at near optimal minimax rate. 
							The related quantile regression methodology will be shown to offer a powerful and yet interpretable generalization of standard linear regression to what one might call a Quantile Linear Model. 
							The QLM, complete with an identification of residual noise, gives a model-based idealization of quantile regression retaining the ability to quantify differential predictor influence on the tails while simultaneously adjusting for noise correlation. 
							I will discuss how QLM leads to a comprehensive inferential framework with the added qualities of model fit assessment and model selection. <br> <br>
							The presentation is based on the following series of papers:<br>
							<span class="dot"></span> - ST Tokdar, S Jiang, EL Cunningham (2024) <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=NQcWoJAAAAAJ&sortby=pubdate&citation_for_view=NQcWoJAAAAAJ:wkm4DBaukwsC">Heavy-tailed density estimation</a>. Journal of the American Statistical Association 119 (545), 163-175 <br>
							<span class="dot"></span> - X Chen, ST Tokdar (2021) <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=NQcWoJAAAAAJ&sortby=pubdate&citation_for_view=NQcWoJAAAAAJ:vxA22ZmNLkoC">Joint Quantile Regression for Spatial Data</a> Journal of the Royal Statistical Society, Series B 83 (4), 826-852 <br>
							<span class="dot"></span> - Y Yang, ST Tokdar (2017) <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=NQcWoJAAAAAJ&sortby=pubdate&citation_for_view=NQcWoJAAAAAJ:r0BpntZqJG4C">Joint estimation of quantile planes over arbitrary predictor spaces</a> Journal of the American Statistical Association 112 (519), 1107-1120 <br>
							<span class="dot"></span> - E Cunningham, ST Tokdar, JS Clark (2020) <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=NQcWoJAAAAAJ&sortby=pubdate&citation_for_view=NQcWoJAAAAAJ:CRzUtm-VnGAC">A vignette on model-based quantile regression: analysing excess zero response</a> Flexible Bayesian regression modelling, 27-64 <br>
						   	<span class="dot"></span> - ST Tokdar, JB Kadane (2012) <a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=NQcWoJAAAAAJ&cstart=20&pagesize=80&sortby=pubdate&citation_for_view=NQcWoJAAAAAJ:QIV2ME_5wuYC">Simultaneous linear quantile regression: A semiparametric Bayesian approach</a> Bayesian Analysis 7 (1), 51-72 
						</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

<!-- Modal Speaker -->
	<div class="modal modal-speaker modal-speaker-5" id="modal-speaker-5" tabindex="-1" role="dialog" aria-labelledby="speaker-5-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Stéphanie van der Pas</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/speakers/speaker-5.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Stéphanie van der Pas</h2>
								<div class="meta">Associate Professor</div>
								<div class="meta mb-2">Vrije Universiteit Amsterdam</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Causal conclusions from a cutoff - Bayesian regression discontinuity designs </p>
						<p class="mb-0">Abstract: With Julia Kowalska and Mark van de Wiel. The regression discontinuity design (RDD) is a quasi-experimental approach used to estimate the causal effects of an intervention assigned based on a cutoff criterion. RDD exploits the idea that close to the cutoff units below and above are similar; hence, they can be meaningfully compared. Consequently, the causal effect can be estimated only locally at the cutoff point. This makes the cutoff point an essential element of RDD. However, especially in medical applications, the exact cutoff location may not always be disclosed to the researcher, and even when it is, the actual location may deviate from the official one. As we illustrate on the application of RDD to the HIV treatment eligibility data, estimating the causal effect at an incorrect cutoff point leads to meaningless results. The method we present, LoTTA (Local Trimmed Taylor Approximation), can be applied both as an estimation and validation tool in RDD. We use a Bayesian approach to incorporate prior knowledge and uncertainty about the cutoff location in the causal effect estimation. At the same time, LoTTA is fitted globally to the whole data, whereas RDD is a local, boundary point estimation problem. In this work we address a natural question that arises: how to make Bayesian inference more local to render a meaningful and powerful estimate of the treatment effect?</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker BR AWARDS!!!!!! -->
	<div class="modal modal-speaker modal-speaker-BR1" id="modal-speaker-BR1" tabindex="-1" role="dialog" aria-labelledby="speaker-BR1-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Fan Bu</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/fan.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Fan Bu</h2>
								<div class="meta">Assistant Professor</div>
								<div class="meta mb-2">University of Michigan</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Data Augmentation Approaches for Inference of Partially Observed Epidemics with Individual Heterogeneity </p>
						<p class="mb-0">Abstract: Understanding the dynamics of infectious disease transmissions requires models that account for both transmission processes and the evolving contact networks underlying them. We propose a generative framework that couples a stochastic epidemic model with a dynamic, adaptive network structure, allowing infection rates to vary with individual-level covariates and network evolution to depend on disease status. This joint process is formulated as a continuous-time Markov chain, capturing the interplay between disease transmission and contact network changes. To address the challenge of inference from partially observed epidemic data, we develop data-augmentation algorithms that efficiently impute missing infection and recovery times while respecting the constraints imposed by the dynamic network structure. Such data augmentation strategy can be implemented through a Markov chain Monte Carlo inference scheme or a stochastic expectation-maximization (EM) algorithm. Through extensive simulations and application to real-world influenza transmission data with high-resolution contact tracing, we demonstrate the efficiency and accuracy of our inference approaches in recovering epidemic parameters with uncertainty quantification. Our framework provides a powerful tool for studying disease transmission on dynamic social networks using modern epidemiological data sources.

</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker BR AWARDS!!!!!! -->
	<div class="modal modal-speaker modal-speaker-BR2" id="modal-speaker-BR2" tabindex="-1" role="dialog" aria-labelledby="speaker-BR2-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Raaz Dwivedi</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/raaz.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Raaz Dwivedi</h2>
								<div class="meta">Assistant Professor</div>
								<div class="meta mb-2">Cornell University</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Advances in Distribution Compression and Its Consequences </p>
						<p class="mb-0">Abstract: This talk will introduce a range of new tools (all part of the python repo <a href="https://github.com/microsoft/goodpoints">goodpoints</a>) for summarizing a probability distribution more effectively than independent sampling or standard Markov chain Monte Carlo thinning in near-linear time. 
							We will also demonstrate how these methods provide provably and practically significant computational speeds ups for a range of machine learning tasks: 
							Monte Carlo Integration, Debiasing Markov Chains, Hypothesis Testing, Kernel Smoothing, Kernel Regression, Attention in transformers, and Stochastic Gradient Methods. <br>
							Based on joint work with  Kyuseong Choi, Carles-Domingo Enrich, Albert Gong, Lingxiao Li, Abhishek Shetty, Lester Mackey. <br>
							Kernel Thinning (<a href="https://arxiv.org/abs/2105.05842">arXiv:2105.05842</a>) <br>
							Generalized Kernel Thinning (<a href="https://arxiv.org/abs/2110.01593">arXiv:2110.01593</a>) <br>
							Distribution Compression in Near-linear Time (<a href="https://arxiv.org/abs/2111.07941">arXiv:2111.07941</a>) <br>
							Compress Then Test: Powerful Kernel Testing in Near-linear Time (<a href="https://arxiv.org/abs/2301.05974">arXiv:2301.05974</a>) <br>
							Debiased Distribution Compression (<a href="https://arxiv.org/abs/2404.12290">arxiv:2404.12290</a>) <br>
							Supervised Kernel Thinning (<a href="https://arxiv.org/abs/2410.13749">arxiv:2410.13749</a>)
						</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker BR AWARDS!!!!!! -->
	<div class="modal modal-speaker modal-speaker-BR3" id="modal-speaker-BR3" tabindex="-1" role="dialog" aria-labelledby="speaker-BR3-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Qian Qin</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/qian.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Qian Qin</h2>
								<div class="meta">Assistant Professor</div>
								<div class="meta mb-2">University of Minnesota</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Spectral gap bounds for reversible hybrid Gibbs chains </p>
						<p class="mb-0">Abstract: Hybrid Gibbs samplers represent a prominent class of approximated Gibbs algorithms that utilize Markov chains to approximate conditional distributions, with the Metropolis-within-Gibbs algorithm standing out as a well-known example. Despite their widespread use in both statistical and non-statistical applications, little is known about their convergence properties. In this talk, I will describe a new type of bound on the convergence rates of  hybrid random-scan Gibbs algorithms. It is shown that the absolute spectral gap of a hybrid Gibbs chain can be bounded based on the absolute spectral gap of the exact Gibbs chain and the absolute spectral gaps of the Markov chains employed for conditional distribution approximations. The bound can be extended to other reversible MCMC algorithms with certain hybrid structures.</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker BR AWARDS!!!!!! -->
	<div class="modal modal-speaker modal-speaker-BR4" id="modal-speaker-BR4" tabindex="-1" role="dialog" aria-labelledby="speaker-BR4-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Giovanni Rebaudo</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/giovanni.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Giovanni Rebaudo</h2>
								<div class="meta">Assistant Professor</div>
								<div class="meta mb-2">University of Turin</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Graph-Aligned Random Partition Model (GARP) </p>
						<p class="mb-0">Abstract: Bayesian nonparametric mixtures and random partition models are powerful tools for probabilistic clustering. However, standard independent mixture models can be restrictive in some applications such as inference on cell lineage due to the biological relations of the clusters. The increasing availability of large genomic data requires new statistical tools to perform model-based clustering and infer the relationship between homogeneous subgroups of units. Motivated by single-cell RNA data we develop a novel dependent mixture model to jointly perform cluster analysis and align the clusters on a graph. Our flexible graph-aligned random partition model (GARP) exploits Gibbs-type priors as building blocks, allowing us to derive analytical results for the probability mass function (pmf) on the graph-aligned random partition. We derive a generalization of the Chinese restaurant process from the pmf and a related efficient and neat MCMC algorithm to implement Bayesian inference. We illustrate posterior inference under the GARP using single-cell RNA-seq data from mice stem cells. We further investigate the performance of the model in recovering the underlying clustering structure as well as the underlying graph by means of simulation studies. </p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker BR AWARDS!!!!!! -->
	<div class="modal modal-speaker modal-speaker-BR5" id="modal-speaker-BR5" tabindex="-1" role="dialog" aria-labelledby="speaker-BR5-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Tommaso Rigon</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/tommaso.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Tommaso Rigon</h2>
								<div class="meta">Assistant Professor</div>
								<div class="meta mb-2">University of Milano-Bicocca</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Bayesian analysis of product feature allocation models </p>
						<p class="mb-0">Abstract: Feature allocation models are an extension of Bayesian nonparametric clustering models, where individuals can share multiple features. We study a broad class of models whose probability distribution has a product form, which includes the popular Indian buffet process. This class plays a prominent role among existing priors, and it shares structural characteristics with Gibbs-type priors in the species sampling framework. We develop a general theory for the entire class, obtaining closed form expressions for the predictive structure and the posterior law of the underlying stochastic process. Additionally, we describe the distribution for the number of features and the number of hitherto unseen features in a future sample, leading to the $\alpha$-diversity for feature models. We also examine notable novel examples, such as mixtures of Indian buffet processes and beta Bernoulli models, where the latter entails a finite random number of features. This methodology finds significant applications in ecology, allowing the estimation of species richness for incidence data, as we demonstrate by analyzing plant diversity in Danish forests and trees in Barro Colorado Island.
								Joint work with: Lorenzo Ghilotti and Federico Camerlenghi.</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->

	<!-- Modal Speaker BR AWARDS!!!!!! -->
	<div class="modal modal-speaker modal-speaker-BR6" id="modal-speaker-BR6" tabindex="-1" role="dialog" aria-labelledby="speaker-BR6-ModalLabel" aria-hidden="true">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-hidden="true">&times;</button>
					<h4 id="speaker-5-ModalLabel" class="modal-title sr-only">Francesca Romana Crucinio</h4>
				</div>
				<div class="modal-body p-0">
					<div class="theme-bg-light p-5">
						<div class="row">
							<div class="col-12 col-md-auto text-center">
							    <img class="profile-image mb-3 mb-md-0 me-md-4 rounded-circle mx-auto" src="assets/images/francesca.jpg" alt="" />
							</div>
							
							<div class="col text-center text-md-start mx-auto">
								<h2 class="name mb-2">Francesca Romana Crucinio</h2>
								<div class="meta">Assistant Professor</div>
								<div class="meta mb-2">University of Turin</div>
							</div><!--//col-->
						</div><!--//row-->
					</div>
					<div class="desc p-4 p-lg-5">
						<p>Title: Towards a turnkey approach to unbiased Monte Carlo estimation of smooth functions of expectations (with applications to exponential random graphs) </p>
						<p class="mb-0">Abstract: Using Monte Carlo samples with expectation m, we develop a general approach to unbiasedly estimate f(m), where f is a smooth function. Specifically, we develop estimators that are based on randomly truncating the Taylor series expansion of f, and estimating the coefficients of the truncated series. We derive their properties, and propose a strategy to set their tuning parameters -- which depend on m -- automatically, with a view to make the whole approach simple to use. We develop our methods for the specific functions f(x)=log x and f(x)=1/x, as they arise in several statistical applications, such as maximum likelihood estimation of latent variable models, and Bayesian inference for un-normalised models. 

</p>
					</div>
				</div><!--//modal-body-->
			</div><!--//modal-content-->
		</div><!--//modal-dialog-->
	</div><!--//modal-->
				
	<!-- Javascript -->          
	<script src="assets/plugins/popper.min.js"></script>
	<script src="assets/plugins/bootstrap/js/bootstrap.min.js"></script>  
	<script src="assets/plugins/smoothscroll.min.js"></script>
	<script src="assets/plugins/gumshoe/gumshoe.polyfills.min.js"></script> 
	<script src="assets/js/main.js"></script>  

</body>
</html> 

